### ISSUE TITLE
EdNet: A Large-Scale Hierarchical Dataset in Education

### ISSUE THREAD

**Issue by hello-im-yj** [One-Day-One-Paper/One-Day-One-Paper#115]
Santa TOEIC 어플리케이션을 통해 수집한 대규모 계층적 데이터셋: EdNet

- 2017~ 2년 이상에 걸쳐 수집
- 784,309명의 학생으로부터 131,417,236개의 상호 작용을 포함
- 현재까지 발표된 가장 큰 공개 IES 데이터 세트

기존 데이터셋과 달리 EdNet은 문제 해결부터 강의 소비, 물품 구매에 이르기까지 다양한 학생 행동을 기록

학생의 학습 활동을 KT 1~4 : 네가지 수준으로 나눈 계층적 구조를 가지고  있음

# 2 EdNet

기존 데이터셋의 한계) ASSISTments[5,15], Junyi Academy[2], Synthetic[16] 및 STATICS[8]와 같은 여러 데이터 세트가 존재하지만, 1) 데이터 기반 모델의 잠재력을 충분히 활용할 만큼 크지 X 2)또한 수집하는 데이터가 question-solving activities로 제한되어 있음. 

![image](https://user-images.githubusercontent.com/63237947/159168445-85bd39b4-f817-49d1-99a2-ca5d87f377c5.png)

**특징**

1. Large-scale : 가장 큰 교육 데이터셋 밴치마크
    1. 평균 441.20개의 상호 작용을 생성
    2. 총 13,169개(95,294,926번의 로그)의 문제와 1,021개의 강의(601,805번의 로그)와 293개의 스킬을 제공 
2. Diversity : 설명읽기 / 강의보기 / 강의 구매 이력 등의 다양한 로그를 제공
3. Hierarchy : KT 1~4 로 갈수록 많은 데이터를 포함
    
    ![image](https://user-images.githubusercontent.com/63237947/159168429-0199452e-63a4-4556-a172-43bad437a9e4.png)

    
    - KT1 : question-response interaction sequences used by most deep knowledge tracing models
    - KT4 : complete list of user actions gathered by Santa.
4. Multi-platform : PC/ 모바일(ios, 안드로이드) 등 다양한 플랫폼을 통한 데이터 수집

# 4 Applications of EdNet

1. Knowledge Tracing
2. Study Session Dropout Prediction in Mobile Learning Environments
3. Pre-training Tasks for Label-scarce Educational Problems
4. RL-based Learning Path Recommendation
